src/main/plugins/org.dita.pdf2/xsl/fo/links.xsl

```diff
@@ -407,9 +406,7 @@ See the accompanying license.txt file for applicable licenses.
           <fo:list-item xsl:use-attribute-sets="related-links.ul.li">
             <fo:list-item-label xsl:use-attribute-sets="related-links.ul.li__label">
               <fo:block xsl:use-attribute-sets="related-links.ul.li__label__content">
-                <fo:inline>
```

Comment:
Is there a reason why we generate an empty `fo:inline` with common attributes, instead of just adding those attributes to the parent `fo:block`. IMO we should stick ID etc. into the parent `fo:block`, if there is a parent element.